structs.go

```diff
@@ -95,6 +95,10 @@ func (Person) TableName() string {
 	return "people"
 }
 
+func (PersonInShort) TableName() string {
+	return "people"
+}
+
```

Comment:
maybe I'm not understanding but does this just return "people" no matter what?

Reply:
This is a required interface for gorm to deduce the table name @kevkevinpal